src/NServiceBus.Core/Scheduling/ScheduledTaskHandlingBehavior.cs

```diff
@@ -0,0 +1,26 @@
+namespace NServiceBus
+{
+    using System;
+    using System.Threading.Tasks;
+    using Pipeline;
+
+    class ScheduledTaskHandlingBehavior : IBehavior<IIncomingLogicalMessageContext, IIncomingLogicalMessageContext>
```

Comment:
Remind me, was there a reason why we prefer IBehavior over https://github.com/Particular/NServiceBus/blob/develop/src/NServiceBus.Core/Pipeline/Behavior.cs ? (performance?)

Reply:
Allocations